.bowerrc

```diff
@@ -1,3 +1,3 @@
 {
-  "directory" : "/_sass"
-}
\ No newline at end of file
+  "directory" : "public/components"
+}
```

Comment:
Changing _sass to public/components looks like an unrelated change. Was there a particular reason why this was changed together with json version?